components/integrations/confirm_integration/confirm_integration.test.tsx

```diff
@@ -31,11 +36,11 @@ describe('components/integrations/ConfirmIntegration', () => {
         user_id: userId,
         display_name: 'bot',
     };
-    const commands = {[id]: {id, token}};
-    const oauthApps = {[id]: oauthApp};
-    const incomingHooks = {[id]: {id}};
-    const outgoingHooks = {[id]: {id, token}};
-    const bots = {[userId]: bot};
+    const commands = {[id]: TestHelper.getCommandMock({id, token})};
+    const oauthApps = {[id]: oauthApp} as unknown as IDMappedObjects<OAuthApp>;
+    const incomingHooks = {[id]: {id}} as unknown as IDMappedObjects<IncomingWebhook>;
+    const outgoingHooks = {[id]: {id, token}} as unknown as IDMappedObjects<OutgoingWebhook>;
+    const bots = {[userId]: bot} as unknown as Record<string, Bot>;
```

Comment:
```suggestion
    const bots: Record<string, Bot> = {[userId]: bot};
```
Instead of casting them as unknown and then to type, why not set a type, I suppose you will need to change bot const to match the type but I think in long run it will be worth it. we have `getBotMock` that can be used.
same goes for the above ones but If it’s seems too much work we can go with current solution

Reply:
@sinansonmez